bbc/simorgh
# WSTEAMA-1186 - Add Billboard a11y swarm doc to storybook

Resolves JIRA [[1186](https://jira.dev.bbc.co.uk/browse/WSTEAMA-1186)]

Overall changes
======

Adds a URL link to a paper doc containing the findings from the a11y swarm to test the Billboard component.

Code changes
======

- Updates the `metadata.json` file for the Billboard component with a url linking to the a11y swarm doc
- Changes the value of done from `false` to `true` 

Testing
======
1. Checkout to the branch and run `yarn storybook`
2. Navigate to the Billboard component
3. Under the README section, the docs should display a green tick next to Accessibility Swarm header and the link should take you to the appropriate paper doc.
